packages/confirm-dialog/src/vaadin-confirm-dialog.js

```diff
@@ -338,7 +341,13 @@ class ConfirmDialog extends ElementMixin(ThemePropertyMixin(ControllerMixin(Poly
       multiple: true,
       observe: false,
       initializer: (node) => {
-        this._messageNodes = [...this._messageNodes, node];
+        const wrapper = document.createElement('div');
+        wrapper.style.display = 'contents';
+        wrapper.id = `confirm-dialog-message-${generateUniqueId()}`;
```

Comment:
nit: we could store the ID here instead of accessing it below.

Reply:
Do you mean something like?

```suggestion
				const wrapperId = `confirm-dialog-message-${generateUniqueId()}`;
        wrapper.id = wrapperId;
```

Reply:
Done.